README.md

```diff
@@ -13,6 +13,13 @@ See also dockerhub tags page: https://hub.docker.com/r/fluent/fluentd-kubernetes
 
 #### Current stable
 
+As the current limitation about the number of automated builds on hub.docker.com, there are some restrictions to ship daemonset images:
+
+* `stackdriver`, `papertrail`, `syslog` images (x86_64/arm64) won't be published anymore
+* `logentries`, `loggy`, `logzio`, `s3` arm64 images won't be published anymore (x86_64 only supported)
+
+If you want to use above non published images, build it by yourself. Dockefile itself is still maintained in this repository.
+
 ##### Multi-Arch images
 - `Azureblob`
   - `docker pull fluent/fluentd-kubernetes-daemonset:v1.13.1-debian-azureblob-1.0`
```

Comment:
loggly and logentries are also not multiarch-ed images.

Reply:
Thanks, fixed.